src/start_server.ts

```diff
@@ -143,6 +150,24 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxyRoot) {
+    const httpProxy = require('http-proxy');
+    const apiProxy = httpProxy.createProxyServer();
+    const proxy = (req: express.Request, res: express.Response) => {
```

Comment:
overloading the `proxy` import, I'm surprised we don't have linting to catch small errors like this